DGP_CVXPY_Code/CS_amp_optimize_BW.py

```diff
@@ -0,0 +1,61 @@
+import cvxpy as cp
+import numpy as np
+
+# Constants
+VTO = 0.5 # Threshold voltage
+KP = 50e-6   # Transconductance parameter (μnCox)
+Cox = 2.3e-3
+mu_n = KP/Cox #Mobility in m^2/Vs
+
+# Given parameters
+Rs = 10e3  # Source resistance in Ohms
+RD = 5e3  # Drain resistance in Ohms
+CL = 1e-12  # Load capacitance in Farads
+
+
+# Create optimization variables
+L = cp.Variable(pos=True)  # Width of the transistor
+VOV = cp.Variable(pos=True) # Gate-source voltage

```

Comment:
It would be nice to have another version where we set VOV=200mV (a reasonable value for design) and have W as the optimization variable. This makes the problem a true sizing optimization (which is the class of problems we are after).